packages/docusaurus-theme-translations/locales/es/theme-common.json

```diff
@@ -3,18 +3,18 @@
   "theme.BackToTopButton.buttonAriaLabel": "Volver al principio",
   "theme.CodeBlock.copied": "Copiado",
   "theme.CodeBlock.copy": "Copiar",
-  "theme.CodeBlock.copyButtonAriaLabel": "Copiar código al portapapeles",
```

Comment:
I don't mind this change, but the original string is:

>   "theme.CodeBlock.copyButtonAriaLabel": "Copy code to clipboard",

This is an aria label for accessibility. Are you sure being less specific will improve accessibility?

Reply:
I've made a small improvement by replacing `Copiar código al portapapeles `with `Copiar código`. This change maintains clarity without affecting accessibility and improves the text for users on different operating systems.